tuf/tuf.go

```diff
@@ -458,8 +491,7 @@ func (tr *Repo) InitRoot(root, timestamp, snapshot, targets data.BaseRole, consi
 	if err != nil {
 		return err
 	}
-	tr.Root = r
-	return nil
+	return tr.SetRoot(r)
```

Comment:
should probably leave as direct assignment given your builder PR is going to take out these `Set___` methods.